src/main/java/org/openpnp/gui/support/IdentifiableObjectToStringConverter.java

```diff
@@ -25,10 +25,7 @@
 public class IdentifiableObjectToStringConverter<T extends Identifiable>
         extends ObjectToStringConverter {
     public String getPreferredStringForItem(Object o) {
-        if (o == null) {
-            return null;
-        }
         T t = (T) o;
-        return t.getId() == null ? "" : t.getId();
+        return t == null || t.getId() == null ? " " : t.getId();
```

Comment:
I thing the `null` object should continue to be converted into the empty string. If I'm not mistaken, there is code testing against `.isEmpty()`.

Reply:
If converted to an empty string then in the combobox it is almost invisible. The space is there intentionaly to add an empty option to the combobox with the same height as other fields. Afaik, this is just the cell renderer that sets the text as a placeholder, it should not have the impact on the value that it is representing but correct me if I am wrong.

Reply:
If it is only used in that context, then OK. But is it? Please check references.

Reply:
I checked and tested during debugging. It is used only in Part/Package/JobPlacement table models and the text is not checked. When the `" "` value is set, the actual value is null as the null object.

Reply:
That's ok then. Thanks for checking.